articles/sso/index.md

```diff
@@ -5,33 +5,46 @@ description: Introduction to Single Sign On (SSO) with Auth0.
 
-![](/media/articles/sso/single-sign-on/sso-checkbox.png)
-
-::: panel-info Note
-For SSO to work you must setup the Identity Provider(s) you wish to use. This involves getting a Client Key and Client Secret from the IdP. Click [here](/identityproviders) to learn how to setup various providers.
+::: panel-info Enabling Identity Providers
+Prior to enabling SSO for a given Client, you must first [configure the Identity Provider(s)](/identityproviders) you want to use.
 :::
 
-This turns the SSO flag is on, this flag can also be set using the API. When this flag is set Auth0 will maintain an SSO session for the user.  This session will last a maximum of 7 days if a user remains active, but will terminate after 3 days of inactivity if the user does not remain active.  To be considered active, a user must access an application registered in Auth0 (the same account as the application that created the session).  
+To enable SSO for one of your Clients (recall that each Client is independent of one another), navigate to the [Clients section of the Auth 0 Management Dashboard](${manage_url}/#/clients). Click on **Settings** (represented by the gear icon) for the Client with which you want to use SSO.
+
+![](/media/articles/sso/single-sign-on/clients-dashboard.png)
+
+Near the bottom of the *Settings* page, toggle **Use Auth0 instead of the IdP to do Single Sign On**.
+
+![](/media/articles/sso/single-sign-on/sso-flag.png)
 
-In addition to turning on the SSO flag in Auth0, applications must add logic to check a user's SSO status. This SSO logic can be implemented client-side with JavaScript or completely server side, click below to learn more:
+You can set the Client's SSO flag using the [Auth0 Management API](/api/management/v2#!/Clients/patch_clients_by_id).
 
-* [Client-side SSO (Single Page Apps)](/sso/single-page-apps-sso)
-* [Server-side SSO (Regular Web Apps)](/sso/regular-web-apps-sso)
+Once you have set the SSO flag in Auth0, you must add logic to your Client to check the user's SSO status. This logic can be implemented either client-side (using JavaScript) or server-side:
 
-> You can see an example of SSO with both Single Page Apps and Regular Web Apps in [this github repository](https://github.com/auth0/auth0-sso-sample)
+* [Client-Side SSO (Single Page Apps)](/sso/single-page-apps-sso)
+* [Server-Side SSO (Regular Web Apps)](/sso/regular-web-apps-sso)
 
+> Please see the [Auth0 SSO Sample](https://github.com/auth0/auth0-sso-sample) repo for an example of SSO with both Single Page Apps and Regular Web Apps.
+
+### Length of SSO Sessions
+
+If the SSO flag is set for a Client, Auth0 will maintain an SSO session for any user authenticating via that Client. If the user remains active, the session will last no more than **7 days**, but if not, the session will terminate after **3 days**. To be considered active, the user must access the Client that created the session within the given timeframe.
 
 # What is Single Log Out?
```

Comment:
@mpaktiti I changed it to ### since it's a subsection of info for SSO, but I can change it back if you think it's more appropriate?